### PR TITLE
5.2.0 release notes

### PR DESCRIPTION
**Pull request by agh1**

### CODE REVIEW

File: release-notes/5.2.0.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+- **(dev/core/91) Search Builder Improvements ([12058](https://github.com/civicrm/civicrm-core/pull/12058))**
+
+- **Rename function enableFullGroupByMode to be reenableFullGroupByMode t… ([12064](https://github.com/civicrm/civicrm-core/pull/12064))**
+
+- **[NFC] FIx indenting in eventFees template ([12062](https://github.com/civicrm/civicrm-core/pull/12062))**
+
+- **Fix for check number no longer exposed dev/core/issues/60 ([12059](https://github.com/civicrm/civicrm-core/pull/12059))**
+
+- **Mark Accent String Tests as incomplete if on a database that is not utf8 ([12060](https://github.com/civicrm/civicrm-core/pull/12060))**
```
Comment (seamuslee001) [inline release-notes/5.2.0.md]: @agh1 I would probably remove this one as its test specific @eileenmcnaughton

**Comment by totten** [comment]
@agh1 are you able to resolve the conflict in `release-notes.md`?

File: release-notes/5.2.0.md [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+- **[dev/core#66](https://lab.civicrm.org/dev/core/issues/66) Standardise
+  Address code in Contact summary report & enable links
+  ([11993](https://github.com/civicrm/civicrm-core/pull/11993))**
+
+  This change refactors the Address related code as part of a drive to get addresses
+  for multiple contacts in one report (relationship), more specifically this
+  adds prefixing into the function used to add the columns. Approach is taken
+  from extended reports.
+
+- **[dev/translation#9](https://lab.civicrm.org/dev/translation/issues/9) Create
```
Comment (eileenmcnaughton) [inline release-notes/5.2.0.md]: this went into 5.1.2 in the end

**Comment by agh1** [comment]
@totten @eileenmcnaughton @colemanw conflict resolved, should be ready to go much thanks to work from @alifrumin.

**Comment by totten** [comment]
Cheers @alifrumin @agh1 !

**Pull request merged**